brave/browser-laptop
# add pref for sync

Adds a preference for sync and categories. Defaults to off.

Test Plan:
1. go to about:preferences#sync and turn sync on
3. kill brave and start it again
4. verify in console that sync is running
5. go to brave.com, verify that no history is synced
6. go to about:preferences#sync, enable syncing history
7. go to brave.com, verify that history is synced this time
8. go to about:preferences#sync, disable sync
9. go to brave.com, bookmark it. verify that nothing is synced at all.


Test Plan:
